test/util/util.go

```diff
@@ -239,8 +239,6 @@ func WaitForBindingReconciledGeneration(client v1beta1servicecatalog.Servicecata
 				return false, fmt.Errorf("error getting ServiceBinding %v/%v: %v", namespace, name, err)
 			}
 
-			fmt.Printf("BINDING: %+v\n\n\n", binding)
```

Comment:
Not in scope for this issue; is an oversight from my last PR. A quick-and-easy fix, so I just threw it in here.